pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -45,6 +45,26 @@ self: super: {
   # Needs older QuickCheck version
   attoparsec-varword = dontCheck super.attoparsec-varword;
 
+  # Fixed at head, but hasn't cut a release in awhile.
+  darcs = doJailbreak (overrideCabal super.darcs (drv: {
+    patches = [
+      # cabal 3.0 support
+      ( pkgs.fetchpatch {
+        url = "https://raw.githubusercontent.com/input-output-hk/haskell.nix/9f8a6e3e5755e4b917686c6267b1135caf724f1f/patches/darcs-setup.patch";
+        sha256 = "sha256-LJDc22EUHchm7hzVVs6LvG115T6OBAI8BdmsGK38GdE=";
+      })
```

Comment:
Could you add a little bit more about what this patch does (and why you're pulling it from haskell.nix and not the darcs upstream)?

Reply:
This patch come from the differences between the 2.14.2 version and the master.